src/LogModel/Microsoft.VisualStudio.ProjectSystemTools.Shared.csproj

```diff
@@ -0,0 +1,37 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\build\import\VisualStudio.props" />
+  
+  <PropertyGroup>
+    <RootNamespace>Microsoft.VisualStudio.ProjectSystem.LogModel</RootNamespace>
```

Comment:
Why isn't this called ProjectSystem.Tools.LogModel or ProjectSystemTools.LogModel?

Reply:
csproj has been changed now